src/main/java/com/exasol/releaserobot/PlatformValidator.java

```diff
@@ -7,5 +9,5 @@
     /**
      * Validate a project.
      */
-    void validate();
+    public void validate(ValidationReport validationReport);
```

Comment:
This is the refactoring candidate for the next step, removing here the output argument and instead calculate it and return it